analyzers/rspec/vbnet/S3871_vb.net.html

```diff
@@ -0,0 +1,30 @@
+<p>The point of having custom exception types is to convey more information than is available in standard types. But custom exception types must be
```

Comment:
I would ague that this should be in the generic part, and that only the compliant and non-compliant should be language specific?